heron/spi/src/java/com/twitter/heron/spi/scheduler/IScalable.java

```diff
@@ -30,8 +31,12 @@
    * Requests new containers for scaling a topology
    *
    * @param containersToAdd Set of containers to be added by the scheduler
+   * @return Map from container-id assigned by packing algorithm, 
+   *             to ContainerPlan with the actual deployed container-id.
```

Comment:
It is good to comment clearly:
- Will the map-to-return include all container-ids to add, no matter whether requesting a remapping? 
Or
- Will the map-to-return include only the container-ids requesting a remapping?
Or
- Will the map-to-return include at least all container-ids requesting a remapping, but also potentially include container-ids not requesting a remapping?

Reply:
updated